### PR TITLE
#2 IfElseTask

### PR DESCRIPTION
**Pull request by KluevJakov**

### CODE REVIEW

File: src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.github.KluevJakov.myinternship.tasks;
+
+public class IfElseTask {
+    public static String solveTask(int n) {
+        if (n%2 != 0) {
```
Comment (kovinz) [inline src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java:5]: please add spaces around '%'

File: src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            return "Weird";
+        } else {
+            if (n >= 2 && n <= 5) {
+                return "Not Weird";
```
Comment (kovinz) [inline src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java:9]: Please move "Not Weird" and "Weird" to constants (private static final String)

File: src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.github.KluevJakov.myinternship.tasks.IfElseTask;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class IfElseTest {
+    @Test
+    public void oddCheck() {
+        assertEquals("Weird", IfElseTask.solveTask(3));
```
Comment (kovinz) [inline src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java:11]: Please move "Not Weird" and "Weird" to constants (private static final String) (copy-paste their declaration from  IfElseTask class)

File: src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Test
+    public void secondIfCheck() {
+        assertEquals("Weird", IfElseTask.solveTask(8));
+    }
+
+    @Test
+    public void elseCheck() {
+        assertEquals("Not Weird", IfElseTask.solveTask(22));
+    }
```
Comment (kovinz) [inline src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java:27]: UTs need to include the end cases in if statements - add tests for numbers: 2 5 6 20

**Review by kovinz — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java (hunk 1 above)
Comment (KluevJakov) [reply src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java]: Done

File: src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package com.github.KluevJakov.myinternship.tasks;
+
+public class IfElseTask {
+    private static final String pos = "Weird";
+    private static final String neg = "Not Weird";
```
Comment (kovinz) [inline src/main/java/com/github/KluevJakov/myinternship/tasks/IfElseTask.java:5]: constants (static final) have contact for naming - name the first one as WEIRD, the second as NOT_WEIRD

File: src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
+package com.github.KluevJakov.myinternship;
+
+import com.github.KluevJakov.myinternship.tasks.IfElseTask;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class IfElseTest {
+    private static final String pos = "Weird";
+    private static final String neg = "Not Weird";
```
Comment (kovinz) [inline src/test/java/com/github/KluevJakov/myinternship/IfElseTest.java:10]: the same naming convention applies here

**Review by kovinz — APPROVED** [review]

**Pull request merged**